docs/admin/admission-controllers.md

```diff
@@ -140,10 +140,31 @@ and ensures that requests in a non-existent `Namespace` are rejected.
 A `Namespace` deletion kicks off a sequence of operations that remove all objects (pods, services, etc.) in that
 namespace.  In order to enforce integrity of that process, we strongly recommend running this plug-in.
 
+### DefaultStorageClass
+
+This plug-in observes creation of `PersistentVolumeClaim` objects that do not require any specific storage class
+and automatically adds a default storage class to them.
+This way, users that do not request any special storage class do no need to care about them at all and they
+will get the default one.
+
+This plug-in does not do anything when no default storage class is configured. When more than one storage
```

Comment:
just few lines below there is "See [persistent volume] documentation about [...] how to mark a storage class as default."

I'd like to have it on one place, as it is probably going to evolve.